changelog.d/20240531_132815_rohan.saeed_supply_mysql8_4_0.md

```diff
@@ -0,0 +1 @@
+- [Improvement] Upgrade MySQL to 8.4.0 (by @rohansaeed)
```

Comment:
This should be labelled as a breaking change :boom:. Please add a note telling people that the upgrade should be automatic for most people, but if they are running a 3rd-party MySQL (RUN_MYSQL=false) then they are expected to upgrade manually.

Reply:
Updated. Please have a look at it again.